discovery-provider/Caddyfile

```diff
@@ -8,11 +8,11 @@ reverse_proxy /comms* comms:8925
 
 reverse_proxy /healthz* healthz
 
+reverse_proxy metabase:{$METABASE_PORT}
+
 route /dashboard* {
     redir /dashboard /dashboard/ 308
     uri strip_prefix /dashboard
     root * /dashboard-dist
     file_server
 }
-
-reverse_proxy backend:5000
```

Comment:
not sure why this was removed? is this intentional?

Reply:
This was to proxy the root to metabase:3000 instead of backend:5000. This would need to be configurable via the plugin being registered or sthg